client/src/App.tsx

```diff
@@ -40,12 +41,13 @@ function App(): JSX.Element {
           <Switch>
             <Route exact path="/login" component={Login} />
             <Route exact path="/signup" component={Signup} />
+            {/* <Route exact path="/calendar" component={MyCalendar} /> */}
```

Comment:
Why do we need to comment this out?

Reply:
I have removed the the route, we are displaying the calendar as a tab.